### DISCUSSION TITLE
Hide search engine - option

### DISCUSSION THREAD

**Discussion by lukaszpolowczyk** [typefully/minimal-twitter#92] [Ideas] [answered]
I don't need the search engine because I use it in "Browse".

In fact, the search engine blinds me:
![obraz](https://user-images.githubusercontent.com/16800535/155188474-0896cb8e-b58f-4ee6-8209-c66ab3af9a29.png)

The search engine needs to disappear because it overlaps the Sort button.

**Comment by ghost** [comment, marked as answer]
Thanks @lukaszpolowczyk, added to [v4.1.8](https://github.com/thomaswang/minimal-twitter/releases/tag/v4.1.8)
